app/javascript/src/components/Dashboard/Notes/utils.js

```diff
@@ -0,0 +1,3 @@
+import dayjs from "utils/dayjs";
```

Comment:
Go through https://academy.bigbinary.com/learn-react/pages/miscellaneous-tips#:~:text=How%20to%20extend%20day.js%3F very well and make necessary changes.

The point of writing and documenting standards in the React Best Practices book is to ensure that we avoid redundant review comments like the one I am giving right now. If you don't agree with what's mentioned in the book, then it's acceptable to keep this change. But you ought to initiate a discussion and make your strong points against keeping the content in the book. If I am convinced then I will be updating the content in the book.

If you feel the content in the book makes sense and if you are not following those standards in your code, then that's not acceptable. I understand that folks miss things out. Thus the `self-review` process. Read: https://handbook.neetokb.com/articles/git-and-github-essentials#self-review-process

Reply:
This was a mistake on my end.   I read it as an example to use dayjs & also I did not find the lib folder under the root folder so I stepped back from creating the lib folder. I will be careful next time.